components/tag-mgt/org.wso2.carbon.identity.tag.manager.api.rest/org.wso2.carbon.identity.tag.manager.api.rest.v1/.openapi-generator-ignore

```diff
@@ -0,0 +1,23 @@
+# OpenAPI Generator Ignore
```

Comment:
Add `.openapi*` to `.gitignore` file and remove .openapi-generator/ files from the PR

Reply:
removed in 52d3387